wiki/Help_Centre/en.md

```diff
@@ -386,58 +464,6 @@ The default values work for most people, but if you find it isn't for you, you c
 - If the default value (0) isn't working for you, try using the `Offset wizard` button to walk you through setting a new one.
 - If this still doesn't help, you may want to try updating your sound card or motherboard audio drivers.
 
-#### I'm experiencing lag spikes or stuttering during play!
```

Comment:
what's the rationale behind deleting this?

Reply:
this paragraph, together w/ the one below ("My user page contents are missing"), is moved to another section -- search the exact text for the addition

Reply:
aha, gotcha